recipes/openldap/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "openldap" %}
+{% set version = "2.4.46" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: ftp://ftp.openldap.org/pub/OpenLDAP/openldap-release/openldap-{{ version }}.tgz
+  sha256: 41975e79406502c092732c57ef0c2c2eb318d91e8e765f81f5d4ab6c1db727c5
+
+build:
+  number: 0
+  skip: True  # [not unix]
+  run_exports:
+    - {{ pin_subpackage('openldap', max_pin='x.x') }}
+  script: 
+    - ./configure --prefix=$PREFIX
```

Comment:
I had to package this at some point... one thing I wish I had done was to split the package into an openldap-client and openldap-server package, as the server comes with a lot of baggage (like BerkeleyDB).

https://www.openldap.org/faq/data/cache/897.html

HAD I BUT KNOWN!

Reply:
Nice find